easybuild/easyconfigs/g/gompi/gompi-1.9.0.eb

```diff
@@ -0,0 +1,24 @@
+easyblock = "Toolchain"
+
+name = 'gompi'
+version = '1.9.0'
+
+homepage = '(none)'
+description = """GNU Compiler Collection (GCC) based compiler toolchain,
+ including OpenMPI for MPI support."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+compname = 'GCC'
+compver = '4.9.2'
+comp = (compname, compver)
+
+# compiler toolchain dependencies
+dependencies = [
+    comp,
+    ('zlib', '1.2.8', '', comp),
```

Comment:
nope, you're right. I can be dropped here. But it is kinda nice to have the full list here?